be/src/exprs/vectorized/string_functions.cpp

```diff
@@ -23,6 +24,17 @@
 
 namespace starrocks::vectorized {
 
+#define THROW_RUNTIME_ERROR_IF_EXCEED_LIMIT(col, func)                         \
+    if (col->reach_capacity_limit()) {                                         \
+        col->reset_column();                                                   \
+        throw std::runtime_error("binary column exceed 4G in function" #func); \
+    }
+
+#define RETURN_COLUMN(col, func)                    \
```

Comment:
suggest to rename func to func_name, otherwise, reader will mistake here should pass a lambda into.

Reply:
good idea